suite/tests/api/dis-armA32.expect

```diff
@@ -4402,7 +4402,7 @@
 +0x44c4   214690ac   smlaltb.cs %r6 %r9 %r12[2byte] %r0[2byte] -> %r6 %r9
```

Comment:
Please add a line `Issue: #3699` or `Fixes #3699` (if it closes the issue) to the PR description.  See https://dynamorio.org/page_code_reviews.html#sec_commit_messages